src/common/templates/check-your-answers.njk

```diff
@@ -13,12 +13,44 @@
         {% endif %}
       </div>
     </dl>
+  
```

Comment:
Moved the macros from the macros.njk to here because it is only used for this screen.